deployments/serializers.py

```diff
@@ -204,6 +213,12 @@ def create(self, validated_data):
         project.save()
         return project
 
+    def update(self, instance, validated_data):
+        project = super().update(instance, validated_data)
+        project.modified_by = self.context['request'].user
+        project.save(update_fields=['modified_by'])
+        return project
```

Comment:
We can also do like this (with single commit)

```suggestion
        validated_data['modified_by'] = self.context['request'].user
        return super().update(instance, validated_data)
```